### DISCUSSION TITLE
After cookies got expired it redirects me to login but on refreshing the login page cookies got set again

### DISCUSSION THREAD

**Discussion by riteshsingla36** [gladly-team/next-firebase-auth#640] [Q&A]
Failed to retrieve the ID token from cookies. This will happen if the user is not logged in, the provided cookie values are invalid, or the cookie values don't align with your cookie settings. The user will be unauthenticated.

i got this in console when cookies got expired but when i refresh the page it sets the cookies again 

<img width="1044" alt="Screenshot 2023-04-21 at 4 22 29 PM" src="https://user-images.githubusercontent.com/85347242/233618510-eebe138d-f84c-4224-945c-1e0235c4af6c.png">

**Comment by elvisplaza** [comment]
i am also getting this error

**Comment by kmjennison** [comment]
Depending on your redirect settings, this could be the expected behavior if the auth cookie is expired (or gets deleted) but the user is still logged in via the Firebase client. The Firebase client serves as the source of truth for authentication, so the auth cookie will be re-set if the user is still logged in. Then, if your login page doesn't use `AuthAction.REDIRECT_TO_APP`, it won't automatically redirect.

If you believe there's a bug, please open a bug issue. Thanks!
